scripts/manage-ruleset.sh

```
@@ -0,0 +1,295 @@
+function print_header() {
+    echo -e "${BLUE}🔐 GitHub Ruleset Manager${NC}"
+    echo "================================"
+    echo ""
+}
+
+function get_ruleset_id() {
+    gh api "repos/$REPO/rulesets" --jq ".[] | select(.name==\"$RULESET_NAME\") | .id" 2>/dev/null
+}
+
+function check_ruleset() {
+    local ruleset_id=$(get_ruleset_id)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)